client/client.go

```diff
@@ -13,6 +14,11 @@ import (
 	"github.com/nspcc-dev/neofs-sdk-go/stat"
 )
 
+const (
+	// max GRPC message size.
+	defaultBufferSize = 4194304 // 4MB
```

Comment:
https://github.com/grpc/grpc-go/blob/bf05b9558c16677e362d231120f8213eb276d406/server.go#L58

But it's not exported, true. Which is _bad_.